cpp/libcudf_kafka/src/kafka_consumer.cpp

```diff
@@ -150,21 +150,29 @@ int64_t kafka_consumer::get_committed_offset(std::string const &topic, int parti
                "Failed retrieve Kafka committed offsets");
 
   int64_t offset = toppar_list[0]->offset();
-  return offset > 0 ? offset : 0;
+  return offset > 0 ? offset : -1001;
```

Comment:
Is this change intentional? Seems unrelated to the API change the PR is about.

Reply:
Yes, we wanted to change the default value for this API, and thought we could just squeeze this in.